src/transformers/models/detr/feature_extraction_detr.py

```diff
@@ -141,6 +141,26 @@ def binary_mask_to_rle(mask):
     return [x for x in runs]
 
 
+def convert_segmentation_to_rle(segmentation):
+    """
+    Args:
+    Converts given segmentation map of shape (height, width) to the run-length encoding (RLE) format.
+        segmentation (`torch.Tensor` or `numpy.array`):
+            A segmentation map of shape `(height, width)` where each value denotes a segment or class id.
```

Comment:
```suggestion
    Converts given segmentation map of shape (height, width) to the run-length encoding (RLE) format.
    
    Args:
            segmentation (`torch.Tensor` or `numpy.array`):
                     A segmentation map of shape `(height, width)` where each value denotes a segment or class id.
```

Reply:
Sorry the tab is a bit messed up on the Github web UI, but you get the point

Reply:
No worries, fixing it now!

Reply:
@NielsRogge could you take a look when you have the time? I'll merge the PR is everything is OK.